app/models/order_article.rb

```diff
@@ -81,6 +81,8 @@ def update_results!
   #      4        |    5     |     4     |           2
   #
   def calculate_units_to_order(quantity, tolerance = 0)
+    quantity = [price.maximum_order_quantity || quantity, quantity].min
+    return 0 if price.minimum_order_quantity && quantity < price.minimum_order_quantity
```

Comment:
I'm not sure if I misunderstand the expected behavior of minimum_order_quantity of this function, or if this case was just missing and just doesn't occur here as it is handled somewhere in client-side already?

Reply:
You're right - at least at a first glance something seems broken here - I've created https://github.com/foodcoopsat/foodsoft_hackathon/issues/99 to address the issue. (I'll take a closer look asap.)